tests/main_test.py

```diff
@@ -45,5 +70,6 @@ async def test_init(mocker: MockerFixture) -> None:
     mocker.patch.object(main, "__name__", "__main__")
     mocker.patch.object(main, 'main', return_value=None)
 
+    #this is broken
     assert await main.init() == None
```

Comment:
As #16 with the huge refactor, we moved from an async init to a synchronous one, it should be enough to remove the `await` and `async`